Services/Migration/DBUpdate_3560/classes/class.ilDBUpdateNewObjectType.php

```diff
@@ -511,94 +526,15 @@ public static function cloneOperation($a_obj_type, $a_source_op_id, $a_target_op
             }
         }
     }
-    
-    /**
-     * Migrate varchar column to text/clob
-     *
-     * @param string $a_table_name
-     * @param string $a_column_name
-     * @return bool
-     */
-    public static function varchar2text($a_table_name, $a_column_name)
-    {
-        global $ilDB;
-        
-        $tmp_column_name = $a_column_name . "_tmp_clob";
-        
-        if (!$ilDB->tableColumnExists($a_table_name, $a_column_name) ||
-            $ilDB->tableColumnExists($a_table_name, $tmp_column_name)) {
-            return false;
-        }
-        
-        // oracle does not support ALTER TABLE varchar2 to CLOB
-
-        $ilAtomQuery = $ilDB->buildAtomQuery();
-        $ilAtomQuery->addTableLock($a_table_name);
-
-        $ilAtomQuery->addQueryCallable(
-            function (ilDBInterface $ilDB) use ($a_table_name, $a_column_name, $tmp_column_name) {
-                $def = array(
-                    'type' => 'clob',
-                    'notnull' => false
-                );
-                $ilDB->addTableColumn($a_table_name, $tmp_column_name, $def);
-
-                $ilDB->manipulate('UPDATE ' . $a_table_name . ' SET ' . $tmp_column_name . ' = ' . $a_column_name);
 
-                $ilDB->dropTableColumn($a_table_name, $a_column_name);
-
-                $ilDB->renameTableColumn($a_table_name, $tmp_column_name, $a_column_name);
-            }
-        );
-
-        $ilAtomQuery->run();
-        
-        return true;
-    }
-    
     /**
-     * Add new RBAC template
-     *
-     * @param string $a_obj_type
-     * @param string $a_id
-     * @param string $a_description
-     * @param int|array $a_op_ids
+     * @param int    $a_rol_id
+     * @param string $a_type
+     * @param array  $a_ops
+     * @param int    $a_ref_id
+     * @return void
+     * @deprecated use Services/AccessControl/classes/Setup/class.ilAccessRolePermissionSetObjective.php instead
      */
-    public static function addRBACTemplate($a_obj_type, $a_id, $a_description, $a_op_ids)
```

Comment:
This leads to several errors in dbupdate_04.php, see https://mantis.ilias.de/view.php?id=32461